app/packages/core/src/components/Grid/useRefreshers.ts

```diff
@@ -35,7 +38,7 @@ export default function useRefreshers() {
     mediaField;
     refresher;
     return uuid();
-  }, [cropToContent, fieldVisibilityStage, mediaField, refresher]);
+  }, [cropToContent, fieldVisibilityStage, mediaField, refresher, schema]);
```

Comment:
@benjaminpkane -- seems like you want to add `schema` to this list of do-nothing expressions?

```
    cropToContent;
    fieldVisibilityStage;
    mediaField;
    refresher;
    schema;
    return uuid();
```

Reply:
Yes, thanks